lib/env_enforcer.rb

```diff
@@ -16,14 +16,20 @@ def call
     end
 
     private def config_file
+      path = root.join('.env_enforcer.yml')
+
       @config_file ||=
-        if File.exist?('.env_enforcer.yml')
-          YAML.load_file('.env_enforcer.yml')
+        if File.exist?(path)
+          YAML.load_file(path)
         else
           fail 'No .env_enforcer.yml file found.'
         end
     end
 
+    private def root
+      Rails.root || Pathname.new(ENV['RAILS_ROOT'] || Dir.pwd)
```

Comment:
isnt `Rails.root` derived from the `RAILS_ROOT` ENV var?  do we need to check for both or can we just do `Rails.root || Dir.pwd`?

also, do we want `Dir.pwd` or do we want `File.dirname(__FILE__)` ?